exporting/exporting_engine.h

```diff
@@ -76,6 +77,10 @@ struct instance_config {
 
 struct simple_connector_config {
     int default_port;
+#ifdef ENABLE_HTTPS
+    SSL *conn; //SSL connection
+    int flags; //The flags for SSL connection
+#endif
```

Comment:
I think this should be in a `struct ..._specific_data`, not in the `struct simple_connector_config`. We also should clean `conn` on Netdata exit (we need to merge #9098 first).

Reply:
I set it here in case it is necessary for other connectors, but no problem, I can move it.

Reply:
The gist is that this is not a configuration, but data.

Reply:
I moved the variable.